### DISCUSSION TITLE
Next version release

### DISCUSSION THREAD

**Discussion by ychartsmatt** [microsoft/vscode-docker#3819] [General]
Hey there, I'm anxiously awaiting the next release of the plugin so that [this bug](https://github.com/microsoft/vscode-docker/issues/2327) is fixed and I can debug Django apps in a container. Is there an estimated date for the next minor release? I've looked all over and not found a schedule. Thanks!

**Comment by bwateratmsft** [comment]
Typically we aim to release on the third Monday of the month. With the really large 1.23.0 release we missed several months, and with the holidays we skipped January, but we should be on track to do our next release on 20 Feb.

In the meantime, if you want you can install a [nightly](https://github.com/microsoft/vscode-docker/issues/1408) build of the Docker extension which would contain that fix.

**Reply by ychartsmatt** [reply]
Awesome, thank you!
